core/node/da_dispatcher/src/metrics.rs

```diff
@@ -0,0 +1,29 @@
+use std::time::Duration;
+
+use vise::{Buckets, Gauge, Histogram, Metrics, Unit};
+
+#[derive(Debug, Metrics)]
+#[metrics(prefix = "server_da_dispatcher")]
+pub(super) struct DataAvailabilityDispatcherMetrics {
+    /// Latency of the dispatch of the blob.
+    #[metrics(buckets = Buckets::LATENCIES, unit = Unit::Seconds)]
+    pub blob_dispatch_latency: Histogram<Duration>,
```

Comment:
it expands the scope of these metrics a bit because the first point in time has to be logged in the component that populates the batch data in the database, and the second one when the blob is dispatched

and this will still not be very meaningful because it will heavily depend on the remaining duration of the sleep for the `da_dispatcher` at the point the batch is ready to be dispatched. e.g. if the `da_dispatcher` just finished the previous iteration, the value of the metric will be close to the 5s(or to the interval set by the operator), but in case it has 0.5s of sleep remaining, the value will be close to the 1s with the same dispatch speed

but if you think this metric will be valuable - I can add it!